_pages/appendix-installing-and-configuring.md

```diff
@@ -13,19 +13,124 @@ development environment for Skyve. These instructions assume that you are
 using Windows and SQL Server. Some changes will need to be made if using a 
 different operating system or database.
 
-If you're new to Java, you may find the instructions in our tutorial easier to follow - [Configure your local environment](https://skyvers.github.io/Aged-care/chapter3/#32-configure-your-local-environment)
+### Download and Install Eclipse and JDK
 
-## Prerequisites checklist
+#### Download and Install Eclipse
 
-Before you begin, ensure you have the following:
+For this section, we have chosen to show how to use Eclipse as the development environment, however, if you prefer, you can use other development environment tools such as IntelliJ, NetBeans, Visual Studio Code, etc. However, if you're not completely familiar with these, use Eclipse so that you can follow the steps below.
 
-* Java 11 JDK - [Temurin OpenJDK](https://adoptium.net/temurin/releases/?version=11) (or an alternative OpenJDK of your choice)
-* Eclipse IDE for Enterprise Java and Web Developers ([www.eclipse.org](https://www.eclipse.org/downloads/packages/))
-    * Note: Do not use the Eclipse IDE for Java Developers
-* Wildfly 26 (select the last final version of Wildfly 26 available) ([http://wildfly.org](http://wildfly.org/downloads/))
-    * Note: Select the Jakarta EE 8 Full & Web Distribution not the EE 9.1 Preview Distribution
-    * Note: Versions of Wildfly > 26 are not supported by Skyve at this time
+NOTE: do not use the Eclipse EE9 preview, you will need the Jakarta EE Full & Web Distribution
+
+  - Click here to [Download Eclipse](https://www.eclipse.org/downloads/packages/)
+  - Next, Click on [Eclipse IDE for Enterprise Java Developers](https://www.eclipse.org/downloads/packages/release/2024-09/r/eclipse-ide-enterprise-java-and-web-developers)
```

Comment:
I think remove this link, as it will get out of date, just keep the name of which Eclipse type they need to download